Pages/Search.cshtml

```diff
@@ -41,10 +43,17 @@
                             </div>
                         </div>
                         <div class="text-center">
-                            <button type="submit" class="btn btn-primary">Search</button>
+                            <button type="submit" class="btn btn-primary btn-lg">Search Breweries</button>
```

Comment:
Enhanced readability for search options 'Search Breweries'

Reply:
Is there a specific accessibility need addressed here?  If so, cite it.  Otherwise, UI changes typically are not included in a code review.  UI is the domain of product owner and designer, which is typically a separate role from developer.